src/wazuh_db/wdb.h

```diff
@@ -2026,6 +2027,17 @@ int wdb_global_get_agent_max_group_priority(wdb_t *wdb, int id);
  */
 wdbc_result wdb_global_assign_agent_group(wdb_t *wdb, int id, cJSON* j_groups, int priority);
 
+/**
+ * @brief Writes the groups of an agent.
```

Comment:
This description doesn't correspond to this method.

Reply:
Fixed